app/routers/audio.py

```diff
@@ -65,14 +66,14 @@ def audio_settings(
 
 @router.post("/settings")
 async def get_choices(
-    session: Session = Depends(get_db),
-    music_on: bool = Form(...),
-    music_choices: Optional[List[str]] = Form(None),
-    music_vol: Optional[int] = Form(None),
-    sfx_on: bool = Form(...),
-    sfx_choice: Optional[str] = Form(None),
-    sfx_vol: Optional[int] = Form(None),
-    user: User = Depends(current_user),
+        session: Session = Depends(get_db),
+        music_on: bool = Form(...),
+        music_choices: Optional[List[str]] = Form(None),
+        music_vol: Optional[int] = Form(None),
+        sfx_on: bool = Form(...),
+        sfx_choice: Optional[str] = Form(None),
+        sfx_vol: Optional[int] = Form(None),
+        user: User = Depends(current_user),
```

Comment:
Please revert. Use git precommit hooks (and disable PyCharm's(?) annoying over-indentations :P )